templates/aws/workspaces.html

```diff
@@ -1,96 +1,156 @@
+
+{% block meta_description %}
+  Ubuntu WorkSpaces offer developers an optimised Ubuntu Desktop environment to rapidly build, test and deploy code &mdash; spinning up and tearing down instances as required. IT organisations benefit from rapid deployment, configuration and scalability whilst their applications and data remain secured in the AWS cloud.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1qC22TM6KZTlxK3CScf-RQ4RUiMUhIBcjSWGur-dbYVA/edit
+{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip u-no-padding--top u-no-padding--bottom">
-  <div class="p-strip--suru-topped is-bordered">
-    <div class="row u-equal-height">
-      <div class="col-7 col-medium-4">
-        <h1>Ubuntu WorkSpaces</h1>
-        <p class="p-heading--3">Virtual desktops, powered by AWS. Develop remotely, securely and cost-effectively.</p>
-        <p>Ubuntu Desktop on Amazon WorkSpaces offers developers an optimised Ubuntu Desktop environment to rapidly build, test and deploy code &mdash; spinning up and tearing down instances as required.</p>
-        <p>IT organisations benefit from rapid deployment, configuration and scalability for their managed Ubuntu Desktops whilst their applications and data remain secured in the AWS cloud, even when their developers are working remotely.</p>
-        <a href="https://aws.amazon.com/workspaces/pricing/?nc=sn&loc=3" class="p-button--positive">Launch Today</a>
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col p-strip is-shallow">
+        <div class="p-section--shallow">
+          <h1>Ubuntu WorkSpaces</h1>
+        </div>
+        <p class="p-heading--5">
+          Virtual desktops, powered by AWS.
+          <br />
+          Develop remotely, securely and cost-effectively.
+        </p>
+        <div class="p-strip is-shallow">
+          OpenSearch is an open source search and analytics suite that developers use to build solutions for search, data observability, data ingestion and more.
+          <br />
+          <br />
+          It is designed for scalability, offering powerful full-text search capabilities and supporting various data types, including structured and unstructured data. OpenSearch has rapidly developed into a standalone platform with unique features and capabilities.
+        </div>
       </div>
-      <div class="col-5 u-align--center u-vertically-center u-align--center u-hide--medium u-hide--small">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/55bb92be-aws_workspaces.png",
-            alt="",
-            height="215",
-            width="603",
-            hi_def=True,
-            loading="auto"
-          ) | safe
-        }}
+      <div class="col p-strip is-shallow">
+        <div class="p-image-container--16-9 is-highlighted">
+          {{ image (url="https://assets.ubuntu.com/v1/b914870e-aws-f7f7f7.png",
```

Comment:
please remove the space between `image` and the bracket, then it should automatically format it correctly

Reply:
Also, we normally do not show the image on small screens in hero sections